### PR TITLE
fix(docs): fix mobile SDK examples, syntax and Description

### PR DESCRIPTION
**Pull request by ghost**
Duplicate of https://github.com/newrelic/docs-website/pull/14333/files to fix CLA issues

### CODE REVIEW

**Review by ndesai-newrelic — APPROVED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 1]
```diff
@@ -1262,7 +1252,7 @@ cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git
             </td>
             <td>
                 ```typescript
-                collectorAddress: "crash-upload-delegate.domain.com"
+                COLLECTOR_ADDRESS = "crash-upload-delegate.domain.com"
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1255]:
```suggestion
                COLLECTOR_ADDRESS="crash-upload-delegate.domain.com"
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 2]
```diff
@@ -1250,7 +1240,7 @@ cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git
             </td>
             <td>
                 ```typescript
-                webViewInstrumentation: true
+                WEB_VIEW_INSTRUMENTATION="true"
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1243]:
```suggestion
                WEB_VIEW_INSTRUMENTATION = "true"
```

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx [hunk 3]
```diff
@@ -1182,7 +1172,7 @@ cordova plugin add https://github.com/newrelic/newrelic-cordova-plugin.git
             </td>
             <td>
                 ```typescript
-                interactionTracingEnabled: true
+                INTERACTION_TRACING_ENABLED="true"
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1175]:
```suggestion
                INTERACTION_TRACING_ENABLED = "true"
```

Comment (homelessbirds) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx:1175]: I don't know if these matter/if you care. Feel free to disregard :)

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx [hunk 4]
```diff
@@ -92,13 +92,13 @@ String sessionId = NewRelic.currentSessionId();
 ### Objective-c [#objc]
 
 ```objectivec
-+ (void) crashNow:(NSString* __nullable)message;
++ (NSString* _Null_unspecified) currentSessionId;;
```
Comment (homelessbirds) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx:95]: are there supposed to be two ;; ?

**Review by homelessbirds — DISMISSED** [review]

File: src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx (hunk 4 above)
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx:95]:
```suggestion
+ (NSString* _Null_unspecified) currentSessionId;
```

Comment (ally-sassman) [reply src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/current-session-id.mdx:95]: Good catch, thank you!

**Pull request merged**